libs/utils/src/project-utils.ts

```
@@ -0,0 +1,266 @@
+export function phaseToFeatureDescription(phase: Phase, milestone: Milestone): string {
+  const parts: string[] = [];
+
+  parts.push(`## ${phase.title}`);
+  parts.push('');
+  parts.push(phase.description);
+
+  if (phase.filesToModify && phase.filesToModify.length > 0) {
+    parts.push('');
+    parts.push('### Files to Modify');
+    parts.push('');
+    phase.filesToModify.forEach((f) => parts.push(`- ${f}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)